Phonebook.Frontend/src/app/app.component.ts

```diff
@@ -154,7 +154,7 @@ export class AppComponent implements OnInit, OnDestroy {
           description: 'Message for just set no cookie url',
           id: 'PageInformationNewUrlNoCookies',
           value:
-            'Save the site URL as a favourite now in order to not get any more dialogs. Please notice: You wont get any information about updates or releases with the set url prameter.'
+            'Save the site URL as a favourite now in order to not get any more startup-dialogs. Please notice: You won\'t get any information about updates or releases with the set url parameter.'
```

Comment:
You could also use these quotes `` ` `` so you don't have to write any special characters in your text: `` `Save the site URL as a favourite now in order to not get any more startup-dialogs. Please notice: You won't get any information about updates or releases with the set url parameter.` ``

Reply:
just for info (no need to commit it, if you don't want to

Reply:
Thanks for your hint, i didn't know that:)